en/docs/learn/configuring-single-sign-on-oidc.md

```diff
@@ -37,6 +37,9 @@ You are now ready to try out OpenID Connect SSO with the Pickup Dispatch and Pic
 2. You will be redirected to the login page of WSO2 Identity Server. Log in using your WSO2 Identity Server credentials (admin/admin). Provide the required consent.
 You will be redirected to the Pickup Dispatch application home page.
 
+    !!! note
```

Comment:
Shall we rephrase this as follows and give it as a note under the "Set up" section so as to warn the user in advance? 

"To avoid errors while redirecting to the home page of the Pickup Dispatch application, upgrade the Tomcat server to the latest version."

Reply:
Done. Thanks for the suggestion! :smiley: